javascript-data-model/order-history.js

```diff
@@ -0,0 +1,43 @@
+
+var orderHistory = [{
+  'Order #1': 'July 3, 2020',
```

Comment:
Property names shouldn't be strings, and they should always start with lowercase. They should be like variables in which they use camelCase.

Although object properties can include illegal variable names, as we saw in our `javascript-objects` Q&A, it's not best practice to include those, especially when we create an object directly. We just wanted to highlight how it's possible for those properties to exist on objects.